packages/mobile/src/components/comments/CommentDrawerForm.tsx

```diff
@@ -9,17 +13,27 @@ import { Box } from '@audius/harmony-native'
 import { CommentForm } from './CommentForm'
 
 export const CommentDrawerForm = () => {
+  const { editingComment, replyingToComment } = useCurrentCommentSection()
   const [postComment, { status: postCommentStatus }] = usePostComment()
+  const [editComment, { status: editCommentStatus }] = useEditComment()
 
   const handlePostComment = (message: string) => {
-    postComment(message, undefined)
+    if (editingComment) {
+      editComment(editingComment.id, message)
+      return
+    }
+
+    postComment(message, replyingToComment?.id)
   }
 
+  const isLoading =
+    (editingComment ? editCommentStatus : postCommentStatus) === Status.LOADING
```

Comment:
Not sure if this looks different on mobile, but I intentionally didnt use the edit status on desktop so that it's just an instant optimistic update

Reply:
Oh yeah good call, I'll change this